packages/css/src/components/card/card.scss

```diff
@@ -67,16 +81,26 @@
   }
 
   &.dso-is-selectable {
-    display: grid;
-    grid-template-columns: 1fr minmax(0, 100%);
+    @extend %card-grid;
 
     .dso-card-selectable {
-      grid-row: span 2;
+      @extend %card-grid-addon;
     }
+  }
+
+  &.dso-has-image {
+    @extend %card-grid;
+
+    .dso-card-image {
+      @extend %card-grid-addon;
 
-    .dso-card-heading,
-    .dso-card-content {
-      grid-column: 2 /-1;
+      margin-right: $u1;
+
+      img {
+        display: flex;
```

Comment:
Ik begrijp niet waarom deze `display: flex` nodig is, in mijn tests lijkt ie niet nodig te zijn.

Verder leuke opbouw met de extends!

Reply:
Hij is nodig omdat je anders een klein stukje whitespace aan de bovenkant van de img ziet. `display: flex` is een veilige manier om die whitespaces te killen.

Reply:
Oki!